scipy/stats/tests/test_stats.py

```diff
@@ -2366,6 +2366,13 @@ def test_skew_propagate_nan(self):
             s = stats.skew(a, axis=1, nan_policy="propagate")
         np.testing.assert_allclose(s, [0, np.nan], atol=1e-15)
 
+    def test_skew_omit_nan(self):
+        # Test that skew returns a scalar when nan_policy is omit
+        # https://github.com/scipy/scipy/issues/12548
+        s = stats.skew([1, 5, 4, 6, np.nan], nan_policy='omit')
```

Comment:
```suggestion
        s = stats.skew([1, 5, 4, 6, np.nan], fisher=True, nan_policy='omit')
```

Reply:
This suggestion is primarily to make explicit that both options for `fisher` arg are tested